influx/queries_per_second.go

```diff
@@ -0,0 +1,77 @@
+package influx
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/influxdata/influxdb/client/v2"
+)
+
+var (
+	queriesPerSecondAggregate = `SELECT count(value) FROM deis_router_response_time_seconds WHERE time > now() - 1m GROUP BY time(1m)`
+	queriesPerSecondPerApp    = `SELECT count(value) FROM deis_router_response_time_seconds WHERE time > now() - 1m AND app = '%s' GROUP BY time(1m)`
+	appQuery                  = `SHOW TAG VALUES FROM deis_router_response_time_seconds WITH KEY = app`
+)
+
+// This code isnt being used right now since we can perform the calculations in grafana. However, we are keeping this around just in case we need it in a future release.
+
+//QueriesPerSecond finds the qps value for the aggregate and individual endpoints
+func QueriesPerSecond(influxClient client.Client) error {
+	err := writeAggregateQueriesPerSecond(influxClient)
+	if err != nil {
+		return err
+	}
+	err = writePerAppQueriesPerSecond(influxClient)
+	if err != nil {
```

Comment:
general note, not blocking for this PR - you can save a line by writing `if err := writePerAppQueriesPerSecond(influxClient); err != nil {`